Bottom: c97a6e5b1aba04fc4916700f219e069d1059c32e
Top:    4de6d8d33b0e95173e59bbcfe4a016fa20735d3f
Author: Ben Greear <greearb@candelatech.com>
Date:   2016-06-06 16:42:45 -0700

Refresh of ath10k-support-more-than-one

---

diff --git a/drivers/net/wireless/ath/ath10k/mac.c b/drivers/net/wireless/ath/ath10k/mac.c
index 4f7a07f..5da9cd7 100644
--- a/drivers/net/wireless/ath/ath10k/mac.c
+++ b/drivers/net/wireless/ath/ath10k/mac.c
@@ -8538,7 +8538,8 @@ int ath10k_mac_register(struct ath10k *ar)
 		}
 		break;
 	case ATH10K_FW_WMI_OP_VERSION_10_4:
-		if (test_bit(ATH10K_FW_FEATURE_WMI_10X_CT, ar->fw_features)) {
+		if (test_bit(ATH10K_FW_FEATURE_WMI_10X_CT,
+			     ar->running_fw->fw_file.fw_features)) {
 			ath10k_10_4_ct_if_comb[0].limits[0].max =
 				ar->max_num_vdevs;
 			ath10k_10_4_ct_if_comb[0].max_interfaces =
@@ -8553,7 +8554,8 @@ int ath10k_mac_register(struct ath10k *ar)
 			ar->hw->wiphy->interface_modes |= BIT(NL80211_IFTYPE_ADHOC);
 
 			/* CT firmware can do tx-sw-crypt if properly configured */
-			if (test_bit(ATH10K_FW_FEATURE_CT_RXSWCRYPT, ar->fw_features) &&
+			if (test_bit(ATH10K_FW_FEATURE_CT_RXSWCRYPT,
+				     ar->running_fw->fw_file.fw_features) &&
 			    ath10k_modparam_nohwcrypt)
 				__clear_bit(IEEE80211_HW_SW_CRYPTO_CONTROL, ar->hw->flags);
 		} else {
